lightning/src/ln/channel.rs

```diff
@@ -1766,7 +1766,21 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			}
 		}
 
-		// The + 1 is for the HTLC that is currently being added to the commitment tx.
+		if self.channel_outbound {
+			// Check that they won't violate our channel reserve by adding this HTLC.
+
+			// One of the +1's is for the HTLC that is currently being added to the commitment tx
+			// and the other is a fee spike buffer we're keeping for the remote (this deviates
```

Comment:
I'm not sure about this, it would prevent an initiator to receives incoming payment, and so increasing its balance and unstucking channel. And it's doesnt work with non-initiator sender policy of keeping one HTLC of buffer?

Reply:
Hmm, this is only for !outbound channels.